src/problem/util/commonDivisor.ts

```diff
@@ -1,3 +1,11 @@
+import { randomInt } from "../util/randomizer";
+
+type Binome = {
```

Comment:
definitely centralize this type definition... this is the 3rd time this is declared ;-)